searx/settings.yml

```diff
@@ -720,25 +720,6 @@ engines:
       requires_api_key: false
       results: JSON
 
-  - name: geektimes
-    engine: xpath
-    paging: true
-    search_url: https://habr.com/en/search/page{pageno}/?q={query}
-    url_xpath: //article[contains(@class, "post")]//a[@class="post__title_link"]/@href
-    title_xpath: //article[contains(@class, "post")]//a[@class="post__title_link"]
-    content_xpath: //article[contains(@class, "post")]//div[contains(@class, "post__text")]
-    categories: it
-    timeout: 4.0
-    disabled: true
-    shortcut: gt
-    about:
-      website: https://geektimes.ru/
-      wikidata_id: Q50572423
-      official_api_documentation:
-      use_official_api: false
-      require_api_key: false
-      results: HTML
-
```

Comment:
In #639 only whalside is named as offline / whats wrong with geektimes.ru? FYI geektimes.ru is redirected to habr.com (see 45f0e1a859fa).

Reply:
We already have a search engine for habr.com ... we don't need two.